tests/ui/test_validation_ui.py

```diff
@@ -43,13 +43,13 @@ def test_dashboard_validation_ui(self, setup_ui):
     @skipif_ui_not_support("validation")
     @skipif_ocs_version("<4.9")
     @black_squad
-    def test_odf_storagesystems_ui(self, setup_ui):
+    def test_odf_storagesystems_ui(self, setup_ui_class):
```

Comment:
Why did you change the name of `setup_ui`?   I don't think it will work, because the function name in conftest file is setup_ui